ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemRootResourceDefinition.java

```diff
@@ -73,7 +73,8 @@
                     .setAllowExpression(true).build();
     static final SimpleAttributeDefinition DEFAULT_RESOURCE_ADAPTER_NAME =
             new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.DEFAULT_RESOURCE_ADAPTER_NAME, ModelType.STRING, true)
-                    .setDefaultValue(new ModelNode().set("hornetq-ra"))
+                    // FIXME this should not be hard-coded
+                    .setDefaultValue(new ModelNode().set("activemq-ra"))
```

Comment:
I added a commit to the PR to transform the ejb3's default-resource-adapter-name for legacy versions. If the value is undefined, it is converted to hornetq-ra.

I bumped the ejb3 management from 3.0.0 to 4.0.0 for this change (leaving the 3.x versions for any changes to WildFly 9.x)